server/lib/builders/document-builder.js

```diff
@@ -23,9 +23,9 @@ const RE_REMOVE_TAGS = /<\/?[^>]*>/gm;
 module.exports = exports = class DocumentBuilder {
 	constructor(content) {
 		this.content = content;
-
+		const standfirst = `<h3>${content.standfirst}</h3>`
```

Comment:
happy for this to be merged but it would be more readable like
const standfirst = content.standfirst ? `<h3>${content.standfirst}</h3>` : null
and then
`<body>${standfirst}${content.bodyHTML}</body>`